auth-web/src/views/auth/AcceptInviteView.vue

```diff
@@ -36,6 +36,8 @@ export default class AcceptInviteView extends Mixins(NextPageMixin) {
   private orgStore = getModule(OrgModule, this.$store)
   private userStore = getModule(UserModule, this.$store)
   private readonly acceptInvitation!: (token: string) => Promise<Invitation>
+  private readonly syncMembership!: (currentAccountId: string) => Promise<Member>
```

Comment:
The actual function in the `org` module takes a number, not a string for the id.

Reply:
Done